ci/jenkins/test.sh

```diff
@@ -853,24 +854,27 @@ function redeploy_k8s_if_ip_mode_changes() {
     HAS_IPV6=${INITIAL_VALUE}
     POD_CIDRS=($( (kubectl get node ${CONTROL_PLANE_HOSTNAME} -o json | jq -r '.spec.podCIDRs | @sh') | tr -d \'\")) || true
     echo "POD_CIDRS=${POD_CIDRS[*]}"
-    for POD_CIDR in "${POD_CIDRS[@]}"; do
-        if [[ $POD_CIDR =~ .*:.* ]]
-        then
-            (( HAS_IPV6++ ))
+    if [[ -n $POD_CIDRS ]]; then
+        for POD_CIDR in "${POD_CIDRS[@]}"; do
+            if [[ $POD_CIDR =~ .*:.* ]]
+            then
+                (( HAS_IPV6++ ))
+            else
+                (( HAS_IPV4++ ))
+            fi
+        done
+        if [[ ${IP_MODE} == "ipv4" ]]; then
+            (( HAS_IPV4-- ))
+        elif [[ ${IP_MODE} == "ipv6" ]]; then
+            (( HAS_IPV6-- ))
         else
-            (( HAS_IPV4++ ))
+            (( HAS_IPV4-- ))
+            (( HAS_IPV6-- ))
+        fi
+        if [ ${HAS_IPV4} -eq ${INITIAL_VALUE} ] && [ ${HAS_IPV6} -eq ${INITIAL_VALUE} ]; then
+          #return 0
+          echo "pass"
```

Comment:
According to the function comment, it will redeploy when the ip mode doesn't match. why we set pass but not `return 0` here?

Reply:
This is debug code to force redeploy K8s. Will remove this.

Reply:
Removed.